static/src/stylesheets/module/atoms/_storyquestions.scss

```diff
@@ -162,3 +162,29 @@
         display: inline;
     }
 }
+
+
+.user__question-title-test {
+    margin-bottom: $gs-baseline * 2;
+}
+
+.user__question-final-thanks {
+    margin-bottom: $gs-baseline;
+}
+
+.delivery-selection-container {
+    display: flex;
+    justify-content: space-between;
+
+    @include mq($until: desktop) {
+        flex-direction: column;
+    }
+}
+
+.user_answer-delivery {
+    flex: 1;
+
+    @include mq($until: desktop) {
+        margin-bottom: 12px;
```

Comment:
This will add a margin to the bottom of the last button too, is that what you want? If not, you can add a _top_ margin to all buttons following another button:
```css
.user_answer-delivery + .user_answer-delivery {
  margin-top: $gs-baseline;
}
```
or, in SASS:
```sass
.user_answer_delivery {
  & + & {
    margin-top: $gs-baseline;
  }
}
```